database/factories/ModelFactory.php

```diff
@@ -67,7 +67,7 @@
     }
 
     return [
-        'sponsor_id' => $sponsor_id,
+        'sponsor_id' => $sponsor_id, // a random sponsor
         'code' => $faker->ean8, // 8 digit barcode
         'currentstate' => 'requested'
```

Comment:
I naturally prefer the latter (becuase IE used to throw a fit with extra commas in JS); but Zend disagrees with me. i'll add commas as per that.